src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -722,6 +723,7 @@ void GNAPlugin::LoadNetwork(const CNNNetwork& _network) {
         manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
         manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
         manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
+        manager.register_pass<ov::intel_gna::pass::MarkupFusableTranspose>();
```

Comment:
Are these specific models (Kaldi) still so important to use such W/A by default ?
Consider to turn this transform off by default. It could be enabled by some flag.

Reply:
> Are these specific models (Kaldi) still so important to use such W/A by default ? Consider to turn this transform off by default. It could be enabled by some flag.

Problem not in old kaldi models only. Other models can contains the same pattern and we will introduce regression from product perspective if removes support

Reply:
I understand,
however IMO we should support valid models by default in the first place not the models which are invalid (or based on some W/A) and not conforming nGraph opset documentation.

We could add information to Release notes, eg: GNA Plugin is extending Transpose support however there is a risk that some models may not work anymore and some flag is needed...

Reply:
> I understand, however IMO we should support valid models by default in the first place not the models which are invalid (or based on some W/A) and not conforming nGraph opset documentation.
> 
> We could add information to Release notes, eg: GNA Plugin is extending Transpose support however there is a risk that some models may not work anymore and some flag is needed...

For kaldi models we can say: yes, it was wrong interpretation of original model by MO (because MO actualy inserts such transposes). But we can't say that customer models is invalid just becuase it uses this specific tranposition.